modules/evm/src/runner/handler.rs

```diff
@@ -227,6 +227,14 @@ impl<'vicinity, 'config, T: Config> Handler<'vicinity, 'config, '_, T> {
 				H256::from_slice(Keccak256::digest(&stream.out()).as_slice()).into()
 			}
 			CreateScheme::Fixed(naddress) => naddress,
+		};
+
+		if address.as_bytes().starts_with(&SYSTEM_CONTRACT_ADDRESS_PREFIX) {
+			Err(DispatchError::Other(
```

Comment:
should just change `DispatchError` to `ExitError` to avoid another convert later